src/services/Wishlist/fetchInitialWishlistService.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const fetchInitialWishlistService = async (token) => {
+  try {
+    const response = await axios.get("/api/user/wishlist", {
+      headers: { authorization: token },
+    });
+
+    if (response.status === 200) {
+    } else {
+      throw new Error();
+    }
+
+    return response;
+  } catch (err) {}
```

Comment:
Same thing as stated above is that you're catching the error but doing nothing with it.

Reply:
Done